--- dnsparser.cc.orig	2009-09-17 16:13:31.514497594 +0000
+++ dnsparser.cc	2009-09-17 16:14:11.585344599 +0000
@@ -20,6 +20,7 @@
 #include "dnswriter.hh"
 #include <boost/lexical_cast.hpp>
 #include <boost/algorithm/string.hpp>
+#include <stdio.h>
 
 using namespace boost;

--- rcpgenerator.cc.orig	2009-09-17 16:14:34.477566910 +0000
+++ rcpgenerator.cc     	2009-09-17 16:14:51.340877985 +0000
@@ -22,6 +22,7 @@
 #include <boost/lexical_cast.hpp>
 #include <iostream>
 #include "base64.hh"
+#include <stdio.h>
 using namespace boost;

 RecordTextReader::RecordTextReader(const string& str, const string& zone) : d_string(str), d_zone(zone), d_po) 
